src/LoginServer/World/Character.cs

```diff
@@ -47,6 +47,7 @@ public Character()
 		{
 			this.Level = 1;
 			this.Channel = 1;
+			this.BarrackLayer = 1;
```

Comment:
This is not directly related to the PR, but should be fixed anyway. It prevents a layer of '0' being saved to the database, and consequently, the character will not show up when `BC_COMMANDER_LIST` is sent.

Reply:
If that's the case BarrackLayer should probably not even accept a 0 if that's not valid. The property could be clamped to 1~max, like I did for HP and co. in my last commit.